src/main/java/uk/gov/pay/webhooks/app/WebhooksModule.java

```diff
@@ -58,8 +60,14 @@ public IdGenerator externalIdGenerator() {
 
     @Provides
     @Singleton
-    public java.net.http.HttpClient netHttpClient() {
-        return java.net.http.HttpClient.newBuilder().connectTimeout(Duration.ofSeconds(5)).build();
+    public Client netHttpClient() {
```

Comment:
Could potentially rename this method, as I think it was referring to `java.net.http.HttpClient` which we're no longer using